package-lock.json

```diff
@@ -7453,6 +7453,12 @@
           "integrity": "sha512-GSmOT2EbHrINBf9SR7CDELwlJ8AENk3Qn7OikK4nFYAu3Ote2+JYNVvkpAEQm3/TLNEJFD/xZJjzyxg3KBWOzw==",
           "dev": true
         },
+        "prettier": {
+          "version": "npm:wp-prettier@1.19.1",
+          "resolved": "https://registry.npmjs.org/wp-prettier/-/wp-prettier-1.19.1.tgz",
+          "integrity": "sha512-mqAC2r1NDmRjG+z3KCJ/i61tycKlmADIjxnDhQab+KBxSAGbF/W7/zwB2guy/ypIeKrrftNsIYkNZZQKf3vJcg==",
+          "dev": true
+        },
```

Comment:
Generally speaking, updates to this file should only happen if it's relevant to the issue. Since this issue is really PHP-specific, this should be left unchanged. I've seen this come up as well and this a simple change so you don't need to revert it; just something to keep in mind for the future 🙂

Reply:
Got it.